src/app/settings/user-billing.component.ts

```diff
@@ -47,7 +47,9 @@ export class UserBillingComponent implements OnInit {
     if (this.organizationId != null) {
       this.billing = await this.apiService.getOrganizationBilling(this.organizationId);
     } else {
-      this.billing = await this.apiService.getUserBilling();
+      // let history = await this.apiService.getUserBillingHistory();
+      // let payment = await this.apiService.getUserBillingPayment();
+      this.billing = new BillingResponse(null);
```

Comment:
What's this?

Reply:
I didn't mean to commit that actually, it was a work around to get the latest jslib in. [This PR ](https://github.com/bitwarden/web/pull/1576) has the real changes we want to have

Reply:
I think we will need to leave this in, merge it, and fix it after the linked PR is merged. We are both dependent on each other to get web to build right now 😵‍💫